### PR TITLE
Add intercom to angular

### PR DESCRIPTION
**Pull request by robguthrie**
ready for review please. Tested and works.

### CODE REVIEW

File: app/serializers/group_serializer.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class GroupSerializer < ActiveModel::Serializer
              :key,
              :name,
              :created_at,
+             :creator_id,
```
Comment (rdbartlett) [inline app/serializers/group_serializer.rb]: Note for another time: I think we actually want this to be something like :account_owner or :org_owner or something. Group creator can leave the group, but there should always be an org owner.

**Comment by rdbartlett** [comment]
Looks good to me but I don't really know what I'm looking at :)

Should be relatively safe to test in production eh?

File: app/views/layouts/angular.html.haml [hunk 2]
```diff
@@ -8,8 +8,7 @@
     %base{href: '/'}
     = stylesheet_link_tag lineman_css_path
 
-    -# google analytics code for production
-    - if Rails.env.production? and ENV['GOOGLE_ANALYTICS_ID']
+    - if ENV['GOOGLE_ANALYTICS_ID']
```
Comment (gdpelican) [inline app/views/layouts/angular.html.haml]: Is this something we want on in dev / test?

**Comment by gdpelican** [comment]
Looks alright to me, although I'm starting to wish didn't need to be using ENV directly in the view for angular.html.haml, and am thus wondering if a `LoomioSettings` class with `intercom_enabled` / `analytics_enabled?` / etc methods should exist.

**Comment by robguthrie** [comment]
I've thought about that a bit and honestly I think it would make things more complex rather than less.

I removed the production lines because of the same thing. If you have those ENV's set then you must want that stuff to run. Why worry about the Rails env too?

**Comment by gdpelican** [comment]
It becomes useful when you want to start having services which have
slightly more complex checks to see if they can function or not (the inline
translation thing would be a good example, which needs a couple different
ENV variables to function correctly), or services which need to appear in
multiple places (less of a problem now that we're running a single page
app, although we'll still have other pages on vanilla rails for a long time
going forward - what if we want to change the criteria for displaying the
Loomio.org footer, but have to do it in a bunch of different templates and
the angular html?

It also makes an easier path towards making an admin panel which would
allow you to toggle features on and off, without having to know anything
about ENV variables.

```
def self.inline_translation_enabled?
  Rails.env.production? &&
  ENV['BING_TRANSLATION_API_KEY'] &&
  ENV['BING_TRANSLATION_SECRET'] &&
  !ENV['TRANSLATION_DISABLED'] # provided trigger from admin panel
end
```

**Comment by robguthrie** [comment]
@gdpelican I've tidied this up a bit.. have a look now.

**Pull request merged**